src/status_im2/contexts/quo_preview/avatars/wallet_user_avatar.cljs

```diff
@@ -1,14 +1,11 @@
 (ns status-im2.contexts.quo-preview.avatars.wallet-user-avatar
-  (:require [quo2.core :as quo]
+  (:require [quo2.components.avatars.wallet-user-avatar.view :as quo]
```

Comment:
Actually we don't want to do this. Please require from quo2.core outside of the quo2 components. We like to treat it like a module